obelix/idefix/main.go

```diff
@@ -2,8 +2,26 @@ package main
 
 import (
 	"fmt"
+	"idefix/redis_management"
+	"log"
 )
 
+// Example for use RedisClient
+func example() {
+	rclient := redis_management.NewRedisClient("localhost:6379")
```

Comment:
Please create an init funtion where you can load the endpoint in a global variable in the client package.